src/status_im2/contexts/wallet/send/transaction_confirmation/view.cljs

```diff
@@ -0,0 +1,136 @@
+(ns status-im2.contexts.wallet.send.transaction-confirmation.view
+  (:require
+    [clojure.string :as string]
+    [quo.core :as quo]
+    [quo.foundations.resources :as quo.resources]
+    [quo.theme :as quo.theme]
+    [react-native.core :as rn]
+    [react-native.safe-area :as safe-area]
+    [reagent.core :as reagent]
+    [status-im2.contexts.wallet.send.transaction-confirmation.style :as style]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(def tabs-data
+  [{:id :tab/assets :label (i18n/label :t/assets) :accessibility-label :assets-tab}
+   {:id :tab/collectibles :label (i18n/label :t/collectibles) :accessibility-label :collectibles-tab}])
+
+(defn- network-names
+  [balances-per-chain]
+  (mapv (fn [chain-id-keyword]
+          (let [chain-id-str (name chain-id-keyword)
+                chain-id     (js/parseInt chain-id-str)]
+            (case chain-id
+              10    {:source (quo.resources/get-network :optimism)}
+              42161 {:source (quo.resources/get-network :arbitrum)}
+              5     {:source (quo.resources/get-network :ethereum)}
+              1     {:source (quo.resources/get-network :ethereum)}
+              :unknown))) ; Default case if the chain-id is not recognized
+        (keys balances-per-chain)))
+
+(defn- asset-component
+  []
+  (fn [token _ _ _]
+    (let [on-press                #(js/alert "Not implemented yet")
+          total-balance           (reduce +
+                                          (map #(js/parseFloat (:balance %))
+                                               (vals (:balancesPerChain token))))
+          total-balance-formatted (.toFixed total-balance 2)
+          currency                :usd
+          currency-symbol         "$"
+          price-fiat-currency     (get-in token [:marketValuesPerCurrency currency :price])
+          balance-fiat            (* total-balance price-fiat-currency)
+          balance-fiat-formatted  (.toFixed balance-fiat 2)
+          networks-list           (network-names (:balancesPerChain token))]
```

Comment:
the sub `:wallet/network-details` will give you this info now. We can make another sub if needed to just get the source, but I have that defined as part of the network details 👍

Reply:
Cool, thanks for the info, this is copy paste code from #17780 so is probably valid on that PR